scripts/exregional_run_prepstart.sh

```diff
@@ -241,6 +249,39 @@ else
     cp_vrfy ${bkpath}/${restart_prefix}fv_srf_wnd.res.tile1.nc    fv_srf_wnd.res.tile1.nc
     cp_vrfy ${bkpath}/${restart_prefix}phy_data.nc                phy_data.nc
     cp_vrfy ${fg_root}/${YYYYMMDDHHmInterv}/${fg_restart_dirname}/INPUT/gfs_ctrl.nc  gfs_ctrl.nc
+
+# do SST update for 01Z onto the restart sfc_data.nc
+    if [ ${HH} -eq ${update_SST} ]; then 
+       echo "Update SST"
+       if [ -r "${SST_ROOT}/latest.SST" ]; then 
+          cp ${SST_ROOT}/latest.SST .
+       elif [ -r "${SST_ROOT}/${YYJJJ00000000}" ]; then 
+          cp ${SST_ROOT}/${YYJJJ00000000} latest.SST
+       else
+         ${ECHO} "${SST_ROOT} data does not exist!!"
+         ${ECHO} "ERROR: No SST update at ${time_str}!!!!"
+       fi
+       if [ -r "latest.SST" ]; then 
+         cp_vrfy ${FIXgsm}/RTG_SST_landmask.dat ./RTG_SST_landmask.dat
+         cp_vrfy ${fg_root}/${YYYYMMDDHHmInterv}/${fg_restart_dirname}/fcst_fv3lam/grid_spec.nc ./fv3_grid_spec
+         cp_vrfy ${FIX_GSI}/${PREDEF_GRID_NAME}/fv3_akbk ./fv3_akbk
+         cat > sst.namelist < EOF
+         &setup
+          bkversion=1,
+          iyear=${YYYY}
+          imonth=${MM}
+          iday=${DD} 
+          ihr=${HH}  
+         /
+         EOF
```

Comment:
This "EOF" should start from the beginning of a line, i.e. no spaces before it. 

Also it will be good to move the whole "cat >...<EOF   ....EOF" part to the beginning of each line.

Reply:
Moved them to beginning of the lines. Thanks!